app/assets/javascripts/sufia/save_work/save_work_control.es6

```diff
@@ -100,5 +101,14 @@ export class SaveWorkControl {
     this.requiredFiles.uncheck()
     return false
   }
+
+  validateAgreement(filesValid) {
+    if (filesValid && this.uploads.hasFiles && this.depositAgreement.mustAgreeAgain) {
```

Comment:
Add `hasNewFiles` which is an alias to `hasFiles`, we'll flesh out the differences in 7.1.0